nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-json-record-utils/src/main/java/org/apache/nifi/json/WriteJsonResult.java

```diff
@@ -48,9 +49,14 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+
+import java.util.regex.Pattern;
 import java.util.function.Supplier;
 
 public class WriteJsonResult extends AbstractRecordSetWriter implements RecordSetWriter, RawRecordWriter {
+    //private static final FieldConverter<Object, String> STRING_FIELD_CONVERTER = StandardFieldConverterRegistry.getRegistry().getFieldConverter(String.class);
```

Comment:
is it expected to have this line as a comment?

Reply:
It's not needed, thank you for pointing out.